ui/admin/app/components/form/group/add-members/index.hbs

```diff
@@ -17,45 +16,38 @@
       @submitText={{t 'resources.group.actions.add-members'}}
       @cancelText={{t 'actions.cancel'}}
     />
-
-    <Rose::Table as |table|>
-      <table.header as |header|>
-        <header.row as |row|>
-          <row.headerCell>{{t 'form.name.label'}}</row.headerCell>
-          <row.headerCell>{{t 'form.id.label'}}</row.headerCell>
-          <row.headerCell>{{t 'resources.scope.title'}}</row.headerCell>
-          <row.headerCell>{{t 'form.description.label'}}</row.headerCell>
-        </header.row>
-      </table.header>
-      <table.body as |body|>
-        {{#each this.availableUsers as |user|}}
-          <body.row as |row|>
-            <row.headerCell>
-              <form.checkbox
-                @label={{user.displayName}}
-                @onChange={{fn this.toggleMember user}}
-              />
-              {{#if user.accountName}}
-                {{!
-                  The following is a formatting hack since we don't have a great
-                  way to manage micro-whitespace or alignment among separate
-                  components without resorting to heavy-handed grids
-                  or contextualization.  This is a one-off.
-                }}
-                <br />
-                &nbsp;&nbsp;&nbsp; &nbsp;&nbsp;
-                <Hds::Badge @text={{user.accountName}} />
+    <Hds::Table
+      @model={{this.availableUsers}}
+      @columns={{array
+        (hash label=(t 'form.name.label'))
+        (hash label=(t 'form.id.label'))
+        (hash label=(t 'resources.scope.title'))
+        (hash label=(t 'form.description.label'))
+      }}
+      @valign='middle'
```

Comment:
Maybe it's just me, but I like the look of the row when it is not vertically aligned here, as it tries to align with the badge from the first cell. (especially with the badge on the third column doesn't sit well with that padding)

This is with @valign='middle'
<img width="1474" alt="Screen Shot 2023-04-25 at 1 25 35 PM" src="https://user-images.githubusercontent.com/15043878/234395694-4a63cc8b-91c3-4a91-b883-db46df9704f4.png">


this is without @valign='middle'
<img width="1359" alt="Screen Shot 2023-04-25 at 1 25 56 PM" src="https://user-images.githubusercontent.com/15043878/234395711-7df35d40-5c7f-4dad-a064-b7bc01c6dc6b.png">

Reply:
I looked all over and couldn't find any guidance from the HDS team, which surprises me a bit but I also don't see any complex examples in Figma. I think aligning to the center makes sense for us 👍 You would only notice the difference with long descriptions and it makes aligning these mixed components much easier.

Reply:
I do normally like to top-align the tables if they are more text heavy, but when you have text with different line-heights the text-baselines get further apart and it gets funky. I have done all sorts of tricks to make it work but I'm not sure we want to do that here. Unless you can think of a way to make them closer

Reply:
I agree it doesn't make much difference to change that, and in this case, middle alignment seems like the right path. Appreciate the feedback!

Reply:
This is a very nice and interesting topic. The classic problem of elements vertical alignment when elements have different baselines.

I agree with @joshuaogle  here, better to keep it as simple as possible since we can get into a rabbit hole trying with vertical alignments.

I just wanted to mention, I am sure `@valign` is an abstraction of a [cell vertical-align](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align). As a suggestion, perhaps we can try to align with value baseline rather than middle? Does that look any better?

I am fine either way, but also I think whatever decision we take, we should be consistent with it, specially thinking on the HDS adoption when we want to replace all the rose table for the HDS ones.

cc: @joshuaogle @DhariniJeeva